include/tvm/relay/dataflow_matcher.h

```diff
@@ -63,7 +66,8 @@ class DFPatternCallbackNode : public Object {
  */
 class DFPatternCallback : public ObjectRef {
  public:
-  TVM_DLL DFPatternCallback(DFPattern pattern, PackedFunc callback, bool require_type);
+  TVM_DLL DFPatternCallback(DFPattern pattern, PackedFunc callback, bool require_type,
+                            bool rewrite_once);
```

Comment:
Given that rewrite once is relatively not a common case, it might be better to make it optional: `bool rewrite_once=false`, so that you don't need to change existing uses in the passes.